internal/transport/transport.go

```diff
@@ -579,7 +579,10 @@ type ClientTransport interface {
 	Close() error
 
 	// GracefulClose starts to tear down the transport. It stops accepting
-	// new RPCs and wait the completion of the pending RPCs.
+	// new RPCs by causing NewStream to return error. Once all streams are
```

Comment:
How about `The transport will stop accepting new RPCs and NewStream will return error`?

Reply:
Done